### PR TITLE
Const issue

### PR DESCRIPTION
**Pull request by sohamThence**
Fix: The constructor being called isn't a const constructor.

### CODE REVIEW

File: example/.flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"sms_autofill","path":"/Users/soham/Projects/packages/sms_autofill/","native_build":true,"dependencies":[]}],"android":[{"name":"sms_autofill","path":"/Users/soham/Projects/packages/sms_autofill/","native_build":true,"dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"sms_autofill","dependencies":[]}],"date_created":"2023-02-20 14:51:17.948304","version":"3.7.3"}
```
Comment (simplenotezy) [inline example/.flutter-plugins-dependencies:1]: You probably want to remove this file

File: example/ios/Flutter/flutter_export_environment.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
+export "FLUTTER_ROOT=/Users/soham/Development/flutter"
```
Comment (simplenotezy) [inline example/ios/Flutter/flutter_export_environment.sh:3]: As well as this - remove the file

File: lib/sms_autofill.dart [hunk 3]
```diff
@@ -70,8 +70,9 @@ class PinFieldAutoFill extends StatefulWidget {
       this.inputFormatters,
       this.enableInteractiveSelection = true,
       this.controller,
-      this.decoration = const UnderlineDecoration(
-          colorBuilder: FixedColorBuilder(Colors.black), textStyle: TextStyle(color: Colors.black)),
+      this.decoration,
+      //  = const UnderlineDecoration(
```
Comment (simplenotezy) [inline lib/sms_autofill.dart:74]: Wonder if it would work up here, but simply removing the "const" in front.

**Review by simplenotezy — CHANGES_REQUESTED** [review]
Minor changes

**Comment by manishkummar21** [comment]
Pls merge this

File: lib/sms_autofill.dart (hunk 3 above)
Comment (sohamThence) [reply lib/sms_autofill.dart]: i was thinking that we keep it nullable, and use null operator to set the default value. made the other changes.

File: example/ios/Flutter/flutter_export_environment.sh (hunk 2 above)
Comment (sohamThence) [reply example/ios/Flutter/flutter_export_environment.sh]: done

File: example/.flutter-plugins-dependencies (hunk 1 above)
Comment (sohamThence) [reply example/.flutter-plugins-dependencies]: done

**Review by LaurentiuSimionescu — APPROVED** [review]

**Review by bontzel — APPROVED** [review]

**Review by Ein6tein — APPROVED** [review]

**Comment by jaumard** [comment]
Thanks for this !

**Comment by LaurentiuSimionescu** [comment]
Is it possible to push this to pub.dev?

**Pull request merged**